### ISSUE TITLE
shad0w doesn't exit from exit command

### ISSUE THREAD

**Issue by Luct0r** [bats3c/shad0w#39]
Issuing the "exit" command doesn't seem to do anything. Currently, Ctrl + C seems to partly get there but errors with threading.py regarding lock.acquire() - after waiting a couple minutes to see if the shutdown process was similar to starting shad0w in terms of length, a second Ctrl + C fully exits.

Tried "exit" twice then a Ctrl + C, and once more to shutdown...
![exitError](https://user-images.githubusercontent.com/43627224/93295460-e27f2480-f7a1-11ea-85a0-2658b4bc2c85.png)

**Comment by bblenard** [comment]
I took a look at this and as far as I can tell after `shad0w` starts up there are 2 threads running: the console thread and the http server thread. While testing I simply changed [the http server thread](https://github.com/bats3c/shad0w/blob/master/shad0w.py#L94-L96) to a daemon thread which according to the [threading library documentation](https://docs.python.org/3/library/threading.html#thread-objects) will cause the program to exit once there are only `daemon` threads executing. This change solves the problem allowing the console thread to exit and then the entire program to exit once the last thread (http server) that is left is a daemon thread. I will open up a PR for this however I am wondering if there was a reason to explicitly set the http server as a non daemon thread?
